plugins/catalog-backend/api-report.md

```diff
@@ -64,9 +66,9 @@ export type AnalyzeLocationEntityField = {
 
 // @public
 export type AnalyzeLocationExistingEntity = {
-  location: LocationSpec;
-  isRegistered: boolean;
-  entity: Entity;
+  target: string;
+  exists: boolean | undefined;
+  entities: CompoundEntityRef[];
```

Comment:
Not sure if I'm allowed to do this change or not, it makes it a lot easier to work on the frontend, but I guess I shouldn't just change public tyes left and right without deprecations or stg. 
I'll explore an option where I'll keep the existing interfaces to solve this.

Reply:
I reverted this change, as it would break the interface and opted for a data transformation on the client side.